x/ibc/core/03-connection/keeper/handshake_test.go

```diff
@@ -691,8 +696,12 @@ func (suite *KeeperTestSuite) TestConnOpenAck() {
 			clientKey := host.FullKeyClientPath(clientB, host.KeyClientState())
 			proofClient, _ := suite.chainB.QueryProof(clientKey)
 
+			// set consensus params in context to simulate baseapp handling
+			ctx := suite.chainA.GetContext()
+			ctx = ctx.WithConsensusParams(suite.chainA.App.GetConsensusParams(ctx))
```

Comment:
Why don't you set the consensus params within the `GetContext` call? Seems better and more DRY since you could easily miss setting in these tests